### DISCUSSION TITLE
Quando começar a estudar javascript?

### DISCUSSION THREAD

**Discussion by 1F1rst** [frontendbr/forum#2488] [Perguntas e Respostas]
eu estou terminando o modulo 3 do curso de html5 e css3 dk Gustavo Guanabara e pelo oq vi o modulo 4 não fala ainda sobre o javascript e o modulo 5 não foi lançado, estou em dúvida se devo terminar este curso e "dominar" o html e css e depois estudar o js e começar a aplicar ao que ja sei ou se devo começar estudar em paralelo, e se devo qual curso de js me recomendam? eu pretendo futuramente comprar algum curso de front-end completo mas não sei qual

**Comment by felquis** [comment]
Eu recomendo o livro "Use a Cabeça JavaScript", muito bom para introduzir o JavaScript e a DOM API para iniciantes, o livro é divertido e de fácil leitura

**Comment by dmchiodini** [comment]
Se você já consegue desenvolver o básicos (coisas simples) utilizando html e css, pode ir para o javascript. Não precisa ser um especialista em html e css, até porque você continuará trabalhando com eles junto do javascript. Recomendo o curso de javascript da B7web, ele é gratuito e tem muitas aulas. 

https://alunos.b7web.com.br/curso/javascript

**Comment by christiancordeiro** [comment]
Continue com o Gustavo Guanabara. Ele tem ótimos cursos, principalmente o de **lógica de programação** que é o ideal para você antes de começar a estudar Javascript. A lógica vai te dar uma base muito boa.
